ui/imports/shared/views/chat/ProfileContextMenu.qml

```diff
@@ -38,9 +38,6 @@ StatusMenu {
     signal removeFromGroup
 
     ProfileHeader {
-        width: parent.width
-        height: visible ? implicitHeight : 0
```

Comment:
No need to set these at all; the `ProfileHeader` has a good implicit size already